ocs_ci/ocs/ui/pvc_ui.py

```diff
@@ -263,7 +317,18 @@ def pvc_clone_ui(
         logger.info(f"Search and select the project {project_name}")
         self.do_click(self.pvc_loc["pvc_project_selector"])
         self.do_send_keys(self.pvc_loc["search-project"], text=project_name)
-        self.do_click(format_locator(self.pvc_loc["test-project-link"], project_name))
+        wait_for_project = self.wait_until_expected_text_is_found(
+            locator=format_locator(self.pvc_loc["test-project-link"], project_name),
+            expected_text=f"{project_name}",
+            timeout=5,
+        )
+        if wait_for_project:
+            logger.info(f"Select test project {project_name}")
+            self.do_click(
+                format_locator(self.pvc_loc["test-project-link"], project_name)
+            )
+        else:
+            logger.error(f"Project {project_name} not found on UI")
```

Comment:
Did you need an exception [if `wait_for_project` is `False`]?

Reply:
Do you want me to raise any specific exception here?

Reply:
Done